app/controllers/api/applications_controller.rb

```diff
@@ -18,7 +18,8 @@ def show
 
   def create
     project = Project.find(application_params['project_id'])
-    if project.reached_application_limit
+    if project.reached_application_limit?
```

Comment:
Render json error

https://medium.com/rails-ember-beyond/error-handling-in-rails-the-modular-way-9afcddd2fe1b